stream_provider.go

```diff
@@ -236,17 +237,29 @@ func (r *streamRegistry) find(streamName string) (*StreamProvider, bool) {
 	return p, ok
 }
 
+func GetFullStreamName(serviceName, streamName string) string {
+	return fmt.Sprintf("%s.%s", serviceName, streamName)
+}
+
+// returns the service name and stream name
+func ParseStreamName(fullStreamName string) (string, string) {
+	li := strings.LastIndex(fullStreamName, ".")
```

Comment:
if you didn't use strings.Split(), it's because you expect to have multiple `.` in the names?

Reply:
Yes, potentially someone could create a service with ´´´.´´´ in the name.
Maybe it's something we should not allow?